lib/views/product/widget/build_product_row.dart

```diff
@@ -5,110 +5,133 @@ import 'package:flipper/utils/flitter_color.dart';
-                  onTap: () {
-                    showDialog(
-                      context: context,
-                      builder: (BuildContext context) {
-                        return AddProductModal(
-                          userId: userId,
-                        );
-                      },
-                    );
-                  },
-                  child: ListTile(
-                    contentPadding: const EdgeInsets.fromLTRB(0, 0, 10, 0),
-                    // leading: callImageBox(context, product),
-                    leading: SizedBox(
-                      height: MediaQuery.of(context).size.height,
-                      width: 58,
-                      child: Container(
-                        decoration:
-                            BoxDecoration(color: HexColor(FlipperColors.gray)),
-                        child: IconButton(
-                          icon: const Icon(
-                            Icons.star_border,
-                            size: 30,
-                          ),
-                          color: Colors.white,
-                          onPressed: () {},
-                        ),
-                      ),
-                    ),
-                    title: Text(
-                      'Reedeem Rewards',
-                      style: TextStyle(color: Colors.black87),
+    Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          GestureDetector(
+            onTap: () {
+              showDialog(
+                context: context,
+                builder: (BuildContext context) {
+                  return AddProductModal(
+                    userId: userId,
+                  );
+                },
+              );
+            },
+            child: ListTile(
+              contentPadding: const EdgeInsets.fromLTRB(0, 0, 10, 0),
+              // leading: callImageBox(context, product),
+              leading: SizedBox(
+                height: MediaQuery
+                    .of(context)
+                    .size
+                    .height,
+                width: 58,
+                child: Container(
+                  decoration:
+                  BoxDecoration(color: HexColor(FlipperColors.gray)),
+                  child: IconButton(
+                    icon: const Icon(
+                      Icons.star_border,
+                      size: 30,
                     ),
+                    color: Colors.white,
+                    onPressed: () {},
                   ),
                 ),
-                Container(
-                  height: 0.5,
-                  color: Colors.black26,
-                ),
-              ]),
-        );
+              ),
+              title: Text(
+                'Reedeem Rewards',
+                style: TextStyle(color: Colors.black87),
+              ),
+            ),
+          ),
+          Container(
+            height: 0.5,
+            color: Colors.black26,

```

Comment:
This will not scale well when we want to implement different themes. please refer on how we do it. @pankajinfowind

Reply:
use something like Theme.of(context).canvasColor